canonical/store-components
# Remove equal height from card list component

## Done
Removed equal height from `CardList` as it was causing problems with the grid

## QA
- View demo or run locally (`yarn docs`)
- Check that the `CardList` component renders in the grid layout
